great_expectations/execution_engine/split_and_sample/sqlalchemy_data_splitter.py

```diff
@@ -494,29 +500,48 @@ def get_split_query_for_data_for_batch_identifiers_for_split_on_date_parts(
         else:
             """
             # NOTE: <Alex>6/29/2022</Alex>
-            Certain SQLAlchemy-compliant backends (e.g., Amazon Redshift) allow only binary operators for "CONCAT".
+            Certain SQLAlchemy-compliant backends (e.g., Amazon Redshift, SQLite) allow only binary operators for "CONCAT".
             """
-            concat_date_parts: concat = sa.func.concat(
-                "",
-                sa.cast(
+            if self._dialect == "sqlite":
```

Comment:
Another case for enums

Reply:
@cdkini it could be done, but I would prefer to do it in a follow-up PR since these values go across many more files and tests than the scope of this PR.

Reply:
I am doing this as part of WIP PR: https://github.com/great-expectations/great_expectations/pull/5980